REFERENCE.md

```diff
@@ -143,49 +143,67 @@ service to prevent `Error: Could not start Service[oddjobd]`.
 
 Default value: []
 
-##### `enable_mkhomedir_flags`
+##### `authconfig_enable_mkhomedir_flags`
```

Comment:
why renaming all these parameters? this makes it a non-backward compatible change.

Reply:
New version resolves this issue.